Source/Tests/FEXBash.cpp

```diff
@@ -33,7 +33,7 @@ int main(int argc, char **argv, char **const envp) {
   auto Args = FEX::ArgLoader::Get();
 
   // Ensure RootFS is setup before config options try to pull CONFIG_ROOTFS
-  if (!FEX::RootFS::Setup(envp)) {
+  if (!FEXServerClient::SetupClient(argv[0])) {
     fprintf(stderr, "RootFS configuration failed.");
```

Comment:
Shouldn't this be updated from "RootFS ..."?

Reply:
Resolved.